FWCore/Services/src/SiteLocalConfigService.cc

```diff
@@ -26,9 +26,36 @@ using namespace xercesc;
 //<<<<<< MEMBER FUNCTION DEFINITIONS                                    >>>>>>
 
 namespace {
+
+  template< class CharType >
```

Comment:
I'm a bit reluctant to create a package for one header file. Both FWCore/Catalog and FWCore/Services would have to depend on it. They do depend on FWCore/Concurrency.